lib/core/CustomRequestHandler.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const logger = require('../util/loggerFactory');
+
+class CustomRequestHandler {
```

Comment:
may be add a reason here why its singleton and why its important here so that nobody changes without asking.

Reply:
Done